spoken/settings.py

```diff
@@ -90,14 +91,16 @@
     'certificate',
     'api',
     'rest_framework',
+    'scriptmanager',
+    'corsheaders',
+    'reversion',
     'workshop',
     'django_filters',
     'impersonate',
-    'corsheaders',
-    'ckeditor'
+    'ckeditor',
+    'rest_framework_swagger'
```

Comment:
swagger api docs are not required in production.

Reply:
ok. but **scriptmanager/urls.py** calls it as shown below
```
from rest_framework_swagger.views import get_swagger_view
schema_view = get_swagger_view(title='Spoken Tutorial\'s API')
```

The API call **http://IP:PORT/scripts/api/docs** will throw an error.

1) Generally, for such cases multiple settings files are created for different environments (dev, prod, etc). But, implementing it at this point may create some issues (in the main system or script system) which I may be unaware of.

2) Instead of deleting the line, I just comment it out and mention a note stating that to use api/docs uncomment the line.

Let me know if the 2nd option is ok with you or any other suggestion that you have.

Reply:
2nd option is fine. Just put a comment on the url.

Reply:
ok

Reply:
Done